src/providers/DevicesProvider/VideoInputProvider.tsx

```diff
@@ -68,14 +68,15 @@ const VideoInputProvider: React.FC = ({ children }) => {
         return;
       }
 
-      const devices = await audioVideo.listVideoInputDevices();
+      const devices = await audioVideo.listVideoInputDevices(true);
```

Comment:
Why do you need to force?

Reply:
Currently, this function is called twice in general:
1. When user joins a meeting without devices. At this time the device info cache with empty label establishes.
2. When user wants to invoke the devices. At this time we need to force update the previous device info cache.

To cover the case 2, I made it force update.
But I need to improve the logic of invoking devices, so the logic may be changed here. I will take a look at it.

Reply:
Removed `forceUpdate` here. Cause the `deviceInfoCache` is updated after setting up new `deviceLabelTrigger`.